extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/runtime/Constituent.java

```diff
@@ -0,0 +1,31 @@
+package io.quarkus.scheduler.runtime;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import jakarta.enterprise.util.Nonbinding;
+import jakarta.inject.Qualifier;
+
+import io.quarkus.scheduler.Scheduled;
+import io.quarkus.scheduler.Scheduler;
+
+/**
+ * This qualifier is used to mark a constituent of a composite {@link Scheduler}, i.e. to distinguish various scheduler
+ * implementations.
+ */
+@Qualifier
+@Documented
+@Retention(RUNTIME)
+@Target({ TYPE, PARAMETER, FIELD })
+public @interface Constituent {
```

Comment:
Personally, I find the name `Constituent` very confusing.
How about something like `@SchedulerImpl`?

Reply:
The name does not really matter because the annotation should never be used outside the runtime module. I chose constituent because the _"a **composite** material is produced from two or more **constituent** materials"_ (wiki). I wanted to avoid anything `@Scheduler*` because the annotation is still on the classpath and therefore users could stumble upon it and wonder... 

In theory, we don't need a specific annotation at all and reuse for example `@io.smallrye.common.annotation.Identifier`.

Reply:
Hm, the `@Identifier` cannot be used because we need a nonbinding value...